### PR TITLE
/deploy: also trigger clang-aarch64 builds when appropriate

### PR DESCRIPTION
**Pull request by dscho**
Let's also build the `clang-aarch64` packages, since we can.

### CODE REVIEW

**Comment by dscho** [comment]
I'll just go ahead and merge this so that I can deploy https://github.com/git-for-windows/MINGW-packages/pull/64 already including the ARM64 package.

**Comment by dscho** [comment]
For the record, this made my heart jump with joy:

image

File: GitForWindowsHelper/slash-commands.js [hunk 1]
```diff
@@ -174,7 +198,10 @@ module.exports = async (context, req) => {
                 )
 
                 const answer = await triggerBuild()
-                const answer2 = await appendToComment(`The workflow run [was started](${answer.html_url})`)
+                const answer2 = await appendToComment(aarch64Answer
+                    ? `The [i686/x86_64](${answer.html_url}) and the [amd64](${aarch64Answer.html_url}) workflow runs were started.`
```
Comment (dennisameling) [inline GitForWindowsHelper/slash-commands.js:202]: Did you mean `arm64` instead of `amd64`? 😊

**Review by dennisameling — COMMENTED** [review]
Exciting stuff! 🚀

File: GitForWindowsHelper/slash-commands.js (hunk 1 above)
Comment (dscho) [reply GitForWindowsHelper/slash-commands.js:202]: 🤦 yes, I did ;-)

**Pull request merged**